packages/components/src/components/Icon/icons.ts

```diff
@@ -25,6 +25,7 @@ export const ICONS = {
     COIN_CONTROL: require('../../images/icons/coin_control.svg'),
     COIN: require('../../images/icons/coin.svg'),
     COINS: require('../../images/icons/coins.svg'),
+    CONFETTI_SUCCESS: require('../../images/icons/confetti.svg'),
```

Comment:
Nit: Please remove `fill="none"` from the SVG file so that it is visible when the SVG viewed outside of Suite.

Reply:
done